tt_metal/llrt/blackhole/bh_hal.cpp

```diff
@@ -69,7 +69,8 @@ void Hal::initialize_bh() {
              (addr < NOC_OVERLAY_START_ADDR + NOC_STREAM_REG_SPACE_SIZE * NOC_NUM_STREAMS)) ||
             ((addr >= NOC0_REGS_START_ADDR) && (addr < NOC0_REGS_START_ADDR + 0x1000)) ||
             ((addr >= NOC1_REGS_START_ADDR) && (addr < NOC1_REGS_START_ADDR + 0x1000)) ||
-            (addr == RISCV_DEBUG_REG_SOFT_RESET_0));
+            (addr == RISCV_DEBUG_REG_SOFT_RESET_0) ||
+            (addr== 0xFFB14000 || addr == 0xFFB14008));  // used to program start addr for eth FW
```

Comment:
Magic numbers? Can we come up with a descriptive name, and make it constexpr variable?

Reply:
will make them consts